api/internal/localizer/localizer.go

```diff
@@ -113,13 +113,39 @@ func (lc *localizer) localize() error {
 // localizeNativeFields localizes paths on kustomize-native fields, like configMapGenerator, that kustomize has a
 // built-in understanding of. This excludes helm-related fields, such as `helmGlobals` and `helmCharts`.
 func (lc *localizer) localizeNativeFields(kust *types.Kustomization) error {
-	for i, path := range kust.Components {
```

Comment:
I might be a little bit lost. Why do we replace Components with OpenAPI? Is this intended?